runner/README.md

```diff
@@ -76,6 +78,29 @@ The `local` backend will submit the run to the locally started shim and runner.
 Continue? [y/n]:
 ```
 
+## Testing remotely
+
+You can also test the built shim and runner using standard backends (including SSH fleets).
+
+> [!NOTE]
+> To run with standard backends, both the runner and shim must be built for linux/amd64.
+
+Build the runner and shim, and upload them to S3 automatically using `just` (see [justfile](../justfile)).
+
+```shell
+just upload
+```
+
+To use the built shim and runner with the dstack server, pass the URLs via `DSTACK_SHIM_DOWNLOAD_URL` and `DSTACK_RUNNER_DOWNLOAD_URL`:
+
+```shell
+DSTACK_SHIM_DOWNLOAD_URL=https://dstack-runner-downloads-stgn.s3.eu-west-1.amazonaws.com/0.0.0/binaries/dstack-shim-linux-amd64
```

Comment:
If multiple people use the same 0.0.0 version, they'll be overriding each other build?